setup.cfg

```diff
@@ -27,7 +27,7 @@ classifiers =
 [options]
 packages = setuptools_rust
 zip_safe = True
-install_requires = semantic_version>=2.6.0; toml>=0.9.0
+install_requires = setuptools>=46.1; semantic_version>=2.6.0; toml>=0.9.0
```

Comment:
Is 46.1 correct? I just copy and pasted the dependency from setup_requires.

By the way ``pyproject.toml`` specifies a different version.

Reply:
As 46.1 is the higher of the numbers, let's use that to be on the safe side.

If you're willing, could you please also update `pyproject.toml` so that they all match?

Reply:
Sure!